README.md

```diff
@@ -40,19 +40,20 @@ Please refer to the [Setting up](#setting-up) section to learn how to set up the
 
 ## Starting the program
 
-**Using Eclipse**
+**Using IntelliJ**
 
-1. Find the project in the `Project Explorer` or `Package Explorer` (usually located at the left side)
-2. Right click on the project
-3. Click `Run As` > `Java Application`
+1. Find the project in the `Project Explorer` (usually located at the left side)
+   1. If the `Project Explorer` is not visible, press <kbd>ALT</kbd>+<kbd>1</kbd> for Windows/Linux, <kbd>CMD</kbd>+<kbd>1</kbd> for macOS to open the `Project Explorer` tab.
+2. Go to the `src` folder and locate the `AddressBook` file.
+3. Right click the file and select `Run AddressBook.main()`
```

Comment:
The full stops are not really consistent (sometimes have, sometimes don't have).